src/components/atoms/Box.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const getSizeClasses = (size) => {
+  switch (size) {
+    case "small": {
+      return "w-[496px] h-[441px]";
+    }
+    case "medium": {
+      return "w-[496px] h-[625px]";
+    }
+    default: {
+      return "w-96 w-10";
+    }
+  }
+};
```

Comment:
```suggestion
const sizeClasses = {
  small: "w-[496px] h-[441px]",
  medium: "w-[496px] h-[625px]",
  default: "w-96 w-10",
};
```
단순하게 정적이라면 이런식으로도 할수 있습니다.

Reply:
한 컴포넌트에 여러 types를 부여해서 사용하는 만큼
용이한 관리를 위해 switch-case문와 object 둘 중 하나로 통일해서 작성해보겠습니다.